crypto/internal.h

```diff
@@ -277,6 +277,34 @@ static inline crypto_word_t constant_time_select_w(crypto_word_t mask,
   return (mask & a) | (~mask & b);
 }
 
+// constant_time_select_8 acts like |constant_time_select| but operates on
```

Comment:
Until this, these constant-time utilities were dead code and so they were removed or not merged in earlier merges. Now we need them, so bring them from the current BoringSSL (we've already merged all changes that BoringSSL has published up to today).